### PR TITLE
Created telemetry infrastructure

### PR DESCRIPTION
**Pull request by davidelorenzoli**
* Instrumented server for Prometheus metrics
* Configured Prometheus server (as Docker image)
* Configured Grafana server (as Docker image) including two sample dashboards

### CODE REVIEW

File: docker/local/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+## Preconditions
+
+It's expected that the image **kaushikchaubal/home-recipes** for container  **home-recipes-server** already exists. Deployment script will use version `latest`.
+
+## Steps
+
+From project root build docker compose
+```
+docker-compose -f docker/local/docker-compose.yml build
```
Comment (kaushikchaubal) [inline docker/local/README.md]: Interesting choice. Why did you decide to use `docker-compose` to spin this up as against creating a k8s Deployment?

File: docker/local/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+```
+From project root build docker compose
+```
+docker-compose -f docker/local/docker-compose.yml up
+```
+
+## Telemetry
+
+As part of the telemetry infrastructure Grafana and Prometheus servers are being created. They are reachable at:
```
Comment (kaushikchaubal) [inline docker/local/README.md:27]: This is great! Good documentation @davidelorenzoli!

**Review by kaushikchaubal — CHANGES_REQUESTED** [review]
@davidelorenzoli - why do we have `docker` as a top level directory. I would suggest to move that under `deploy/local`

File: server/recipes/main.go [hunk 3]
```diff
@@ -1,87 +1,66 @@
...
+const metricsPort = ":2112"
 
-type server struct{}
-
-func (s *server) AddRecipe(ctx context.Context, req *generated.AddRecipeRequest) (*generated.AddRecipeResponse, error) {
-	log.Printf("Adding new recipe\t\tName: %v, Cuisine: %v", req.GetRecipe().GetName(), req.GetRecipe().GetCuisine())
-	return &generated.AddRecipeResponse{Success: true}, nil
-}
+func main() {
+	bootstraps := []interface{}{
```
Comment (kaushikchaubal) [inline server/recipes/main.go:18]: Nice change!

**Comment by kaushikchaubal** [comment]
Overall, the changes look good. Once this comment (https://github.com/kaushikchaubal/home-recipes/pull/2#pullrequestreview-404889993) is addressed, I can go ahead and merge this PR.

File: docker/local/README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# Docker Deployment
```
Comment (kaushikchaubal) [inline docker/local/README.md]: Good documentation @davidelorenzoli!

File: docker/local/README.md (hunk 1 above)
Comment (davidelorenzoli) [reply docker/local/README.md]: The Docker local is for testing only, you don't need the whole k8s war machine for that. Just to keep things simple.

**Comment by davidelorenzoli** [comment]
> @davidelorenzoli - why do we have `docker` as a top level directory. I would suggest to move that under `deploy/local`

Done

**Review by kaushikchaubal — APPROVED** [review]

**Comment by kaushikchaubal** [comment]
lgtm! 🥇

**Pull request merged**